README-zh.md

```diff
@@ -0,0 +1,84 @@
+# Open Platform for AI (PAI) ![alt text][logo]
+
+[logo]: ./pailogo.jpg "OpenPAI"
+
+[![Build Status](https://travis-ci.org/Microsoft/pai.svg?branch=master)](https://travis-ci.org/Microsoft/pai)
+[![Coverage Status](https://coveralls.io/repos/github/Microsoft/pai/badge.svg?branch=master)](https://coveralls.io/github/Microsoft/pai?branch=master)
+
+
+## 概述
```

Comment:
>概述 [](start = 3, length = 2)

please exclude the top level readme from this PR and focus on webportal only. Also, the readme has been revised, please update the translation accordingly.

Reply:
OK, and I noticed your update, this is already new translation.